src/services/policy.ts

```diff
@@ -30,9 +30,9 @@ const SET_POLICY_EVENT_SELECTOR =
   '0xa79c31a86c9b0b2abf73ad994711fbad4da038921b96087ff074964aecc528';
 
 const network = process.env.NETWORK as constants.StarknetChainId;
-const nodeUrl = process.env.NODE_RPC_URL;
+const nodeUrl = process.env.NODE_RPC_URL!;
```

Comment:
ℹ️ non-blocking as I don't have the solution

Weird, you shouldn't have to do this.
Our environment variables are typed in [src/types/NodeJS.d.ts](https://github.com/LedgerHQ/StarkCheck/blob/main/src/types/NodeJS.d.ts). tsc and ts-node load them and understand them perfectly that way (that's why the build command works without this operator). 

Meaning `nodemon` doesn't load custom types, I didn't know that. I'm wondering if `ts-node-dev` solves this or if there is a flag to pass to nodemon to make it iso tsc/ts-node. In a perfect world, our dev and prod env would be iso'

Reply:
+ not a big deal but this operator triggers a warning with our current eslint configuration.